input.txt

```diff
@@ -0,0 +1,5 @@
+! TRAVIS input file
```

Comment:
We shouldn't need this file; we should either remove it or .gitignore it (you'll also need to remove it from the cache if the latter, with `git rm --cached input.txt)